corehq/apps/accounting/utils.py

```diff
@@ -78,7 +78,7 @@ def fmt_product_rate_dict(product_name, product_rate=None):
             product_rate = SoftwareProductRate.objects.create(name=product_name, is_active=True)
     return {
         'name': product_rate.name,
-        'product_type': SoftwareProductType.COMMCARE,
+        'product_type': 'CommCare',  # TODO - kill product_type
```

Comment:
What's holding you back from killing this? It (maybe naively) seems like you could just remove all references to this?

Reply:
It gets used somewhere in JS and killing it breaks the internal accounting pages.  I tried tracking this down but got nowhere - leaving as an exercise for later.